src/Campaigns/resources/admin/components/CampaignsListTable/index.tsx

```diff
@@ -0,0 +1,97 @@
+import {__} from '@wordpress/i18n';
+import {ListTablePage} from '@givewp/components';
+import ListTableApi from '@givewp/components/ListTable/api';
+import {BulkActionsConfig, FilterConfig} from '@givewp/components/ListTable/ListTablePage';
+import {IdBadge} from '@givewp/components/ListTable/TableCell';
+import {Interweave} from 'interweave';
+import {CampaignsRowActions} from './CampaignsRowActions';
+import styles from './CampaignsListTable.module.scss';
+import {GiveCampaignsListTable} from './types';
+import CreateCampaignModal from '../CreateCampaignModal';
+
+declare global {
```

Comment:
Let's use a bit different approach here. 
```ts
declare const window: {
    GiveCampaignsListTable
} & Window;
```

The problem with `declare global` is that all projects opened in the IDE will reference this. Let's keep this local.

Reply:
Resolved in c09555969c69e626938f96c89b83ecaee94a4bd3 and ed96921e53a00205882df49881b9c6aac5f5995d